src/main/java/org/broadinstitute/barclay/argparser/CommandLinePluginDescriptor.java

```diff
@@ -161,7 +161,7 @@ public abstract Object getInstance(Class<?> pluggableClass)
      * @return the default plugins used for this instance of this descriptor as a list of Object. Used for
      * help/doc generation.
      */
-    public abstract List<Object> getDefaultInstances();
+    public abstract List<? extends T> getDefaultInstances();
```

Comment:
@cmnbroad & I both think this should be `? super T`, since there's client code that will want this typed as `List<Object>`, and there's also client code that could want to type it as `List<T>`, but it seems unlikely that there would be client code that would type it as a List of a subclass of T.